recipeapp/src/Recipes/index.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import SearchBar from './SearchBar/SearchBar';
+import '../Recipes/recipes.css'
+
+class Recipes extends Component {
```

Comment:
If you're feeling adventurous, you could create a new `root` directory that houses this component.

```
recipes
    | - Root
    |     | - Root.js
    |     | - Root.css
    |     | - index.js
    | - SearchBar 
    |     | - SearchBar.js
    |     | - SearchBar.css
    |     | - index.js
    | - index.js
```
The `index.js` file within the recipe folder's root could then be simplified to something like this:
```JavaScript
export { SearchBar } from './SearchBar';
export { Root as Recipes } from './Root';
```

Reply:
I have become adventurous and worked on this